cmd/gobl/bulk.go

```diff
@@ -31,7 +31,7 @@ func (o *bulkOpts) runE(cmd *cobra.Command, args []string) error {
 	if o.indent {
 		enc.SetIndent("", "\t")
 	}
-	for result := range internal.Bulk(ctx, in) {
+	for result := range internal.Bulk(ctx, nil, in) {
```

Comment:
@samlown what do you think about this? Should I live it like this or could I delete the file?

Reply:
oh, I see... I guess this was to enable a JSON interface from the command line? Perhaps it was never finished. In my opinion, this could be useful, if it works! I'd leave it in for the time being. If it's easy to add a new commend, that'd be great to test with. I guess something like `gobl bulk`?

Reply:
ok, leaving it for the moment